### PR TITLE
Classified z logo and favicon

### PR DESCRIPTION
**Pull request by monnyy**
This solves issues #116 and #117

### CODE REVIEW

**Review by kevcamel — COMMENTED** [review]
It is best practice to use **em** instead of **px** on ine 143 of the CSS. 1 EM is equal to 1x the relative font size of the device, whil 1px is always = 1px.

https://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/

**Comment by kevcamel** [comment]
screen shot 2017-10-16 at 12 59 09 am
screen shot 2017-10-16 at 12 59 16 am
i dunno about u guys but i really like it

File: public/css/stylesheet.css [hunk 1]
```diff
@@ -139,8 +140,7 @@ Aligns Main Menu under button
   height:50px;
   position:relative;
   left:1.1%;
-  font-size:40px
-  margin-top:10;
+  font-size:40em;
```
Comment (kevcamel) [inline public/css/stylesheet.css:143]: what exactly is this doing?  the pannel headings are our "login" and "register buttons".. I dont think we need to change it when the H1 tag is already chagning the font size?

Comment (neqqash) [reply public/css/stylesheet.css:143]: the margin-top is useless, i was sure it was taken out.

**Review by kevcamel — COMMENTED** [review]
Code and comments look good. The new logo looks even better.

**Comment by kevcamel** [comment]
waiting for travis to complete

**Pull request merged**